src/Codeception/Lib/Generator/Actor.php

```diff
@@ -30,10 +51,19 @@ class {{actor}} extends \Codeception\Actor
 
 EOF;
 
+    /**
+     * @var string
+     */
     protected $inheritedMethodTemplate = ' * @method {{return}} {{method}}({{params}})';
 
     protected $settings;
```

Comment:
Based on usage and changes made in other generators, $settings must be an array, right?